app/api/events.py

```diff
@@ -128,19 +183,29 @@ def after_create_object(self, event, data, view_kwargs):
         uer = UsersEventsRoles(user, event, role)
         save_to_db(uer, 'Event Saved')
 
-    decorators = (accessible_events, )
+    def before_create_object(self, data, view_kwargs):
+        data['starts-at'] = data['starts-at'].replace(tzinfo=None)
```

Comment:
@poush your database might not be updated , also try to create new events and then again try fetching them , I had some errors with old events but after creating new events it panned out well.